### DISCUSSION TITLE
Help! Friesach does not load after setting up Crescendo

### DISCUSSION THREAD

**Discussion by Aduragbemi-Adeyinka** [GrandOrgue/grandorgue#1699] [Help]
I'd love to use the Crescendo for Friesach sampleset. But, after saving the registrations for all the 32 steps, Friesach would not load, and GrandOrgue would crash. The only remedy that I figured out was to delete the Data and Cache files, then Friesach would load like brand new with all configurations lost. 

This did not occur when I used the Demo sampleset. 

I have programmed the Crescendo successfully before, on my old VPO, with GrandOrgue version 3.8.xxxx, where the Override function was first released, and this issue did not occur then. 

Could this be a kind of bug in the later versions?

**Comment by thbeckpr** [comment]
I'm not one of the developers, but since no one has answered, I'll assume you have taken the obvious step of installing the latest version. Have you tried to revert to an earlier version to see if it still works correctly?

**Comment by Aduragbemi-Adeyinka** [comment]
Yes I loaded v3.9.4-1 and I have not had any problem yet. Time will tell.

**Reply by Aduragbemi-Adeyinka** [reply]
A fresh problem with this version - it seems I'm unable to save registrations to the pistons. I'm certain that this has been solved in later versions, so I will continue searching for the version that will give us the required functions without having any issue.

**Reply by Aduragbemi-Adeyinka** [reply]
v3.8.0-1 has been OK for me. No issues yet. I'm going to stay on this version for a while
